### ISSUE TITLE
cwtv.com

### ISSUE THREAD

**Issue by kalbasit** [reek/anti-adblock-killer#762]
The current handling for [cwtv](https://github.com/reek/anti-adblock-killer/blob/aa0937c94370af9276e472f13ad417918e62cfa3/anti-adblock-killer.user.js#L2008-L2020) looks like it's not working anymore. The HTML page included the following code before:

```
$(document).ready(function() {
   if (typeof CLAPI != 'undefined') {
                CLAPI.IsAdBlocking(function(isAdBlocking, hasAdBlocked) {
                        // respond to ad block state
                        //console.log('IsAdBlocking ' + isAdBlocking); console.log('hasAdBlocked ' + hasAdBlocked);
                        if (isAdBlocking) {
                            $('#videocontainerwrap').html("<h4><p>Oops, you have an ad blocker enabled!</p><p>To watch this episode, please disable it and refresh the page.</p></h4>");
                    }
                });
        }    
});
```

That code has been broken into two parts now:

First part:

```
<script>
 	function CWTVIsAdBlocking(isAdBlocking) {
		// respond to ad block state
		if (isAdBlocking) {
		    $('#videocontainerwrap').html("<h4><p>Oops, you have an ad blocker enabled!</p><p>To watch this episode, please disable it and refresh the page.</p></h4>");
	    }
    }
</script>
```

Second parts

```
<script>

if (typeof CWTVIsAdBlocking != 'undefined') {
    setTimeout(function() {
        var c = 1,
            adel = document.getElementById('ad_728x90');
        if (adel) {
            c = 0;
            if (adel.offsetParent === null
        		|| adel.offsetHeight == 0
        		|| adel.offsetLeft == 0
        		|| adel.offsetTop == 0
        		|| adel.offsetWidth == 0
        		|| adel.clientHeight == 0
        		|| adel.clientWidth == 0) {
                c = 1;
            }
    		if (c == 0 && window.getComputedStyle !== undefined) {
    			var t = window.getComputedStyle(adel, null);
    			if (t.getPropertyValue('display') == 'none'
    			    || t.getPropertyValue('visibility') == 'hidden') {
    				c = 1;
    			}
    		}
    		if (c == 0) {
                if (adel.childNodes.length > 0 && 4 > adel.childNodes.length) {
                    var i = adel.childNodes.length;
                    while (i--) {
                        var n = adel.childNodes[i];
                        if (n.nodeName.toLowerCase() == 'script') {
                            if (n.innerHTML.indexOf('googletag.cmd.push') !== false) {
                                c = 1;
                                break;
                            }
                        }
                    }
                }
            }
        }
        CWTVIsAdBlocking(c);        
    }, 10000);
}
</script>
```

**Comment by reek** [comment]
HI,

Ok thanks, can you try this fix

<pre>
    cwtv_com : {
      // issue: https://github.com/reek/anti-adblock-killer/issues/762
      // issue: https://github.com/reek/anti-adblock-killer/issues/340
      // code: http://pastebin.com/J7e73MpJ
      host : ['cwtv.com'],
      onStart : function () {
        Aak.uw.CLAPI = undefined;
        Aak.uw.CWTVIsAdBlocking = undefined;
      },
      onBeforeScript : function (e) {
        if (Aak.contains(Aak.innerScript(e), '<h4><p>Oops, you have an ad blocker enabled!</p><p>To watch this episode, please disable it and refresh the page.</p></h4>')) {
          Aak.stopScript(e);
        }
      }
    },
</pre>

**Comment by kalbasit** [comment]
Hi @reek #763 worked for me but let me try yours as well.

**Comment by kalbasit** [comment]
no it didn't work.

**Comment by reek** [comment]
Hi,<br><br>Anti-AdBlock Killer has been updated ! <a href="https://github.com/reek/anti-adblock-killer#changelog">Changelog</a><br><br>Step 1: <a href="http://tinyurl.com/ong3bfe">Update</a> : UserScript<br>Step 2: <a href="http://reek.github.io/anti-adblock-killer/#fiterlist">Update</a> : FilterList<br><br>Do not forget to donate, review, share, follow and add to favorites.<br><br>See you<br>Reek

https://github.com/reek/anti-adblock-killer/
https://twitter.com/antiadbkiller

**Comment by kalbasit** [comment]
fixed in 657b64dc334975912351a9d2b9004f3eadbee42d
